app/controllers/MissionController.scala

```diff
@@ -37,7 +37,22 @@ class MissionController @Inject() (implicit val env: Environment[User, SessionAu
         }
 
         val completedMissions: List[Mission] = MissionTable.selectCompletedMissionsByAUser(user.userId)
-        val incompleteMissions: List[Mission] = MissionTable.selectIncompleteMissionsByAUser(user.userId)
+        val regionsWhereOriginal1000FtMissionCompleted = completedMissions.filter(m => 
+          m.coverage match {
+            case None => ~=(m.distance_ft.getOrElse(Double.PositiveInfinity), 1000.0, 0.10)
```

Comment:
For these constants (distance, precision, etc), define vals for those above the code where they are used.